### PR TITLE
POWR-3873 Update PHP to 8.0 in Pantheon and Lando

### PR DESCRIPTION
**Pull request by kxwang**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -5,23 +5,21 @@
...
         username: egovpdx
         password: $DOCKERHUB_PASSWORD
-  working_directory: /var/www/html
+  working_directory: /home/circleci/app
   steps:
     - restore_cache:
         name: Retore Git cache before checkout
         keys:
-          - source-v1-{{ .Branch }}-{{ .Revision }}
-          - source-v1-{{ .Branch }}-
```
Comment (RichardDavies) [inline .circleci/config.yml:18]: Why did you remove the fallback keys?

Comment (RichardDavies) [reply .circleci/config.yml:18]: Their documentation examples include fallback keys just like these so I'm curious why you removed them. https://circleci.com/docs/2.0/caching/

File: .circleci/config.yml [hunk 2]
```diff
@@ -5,23 +5,22 @@
...
-          - source-v1-
```
Comment (RichardDavies) [inline .circleci/config.yml:19]: You missed restoring this cache key.

**Review by RichardDavies — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (kxwang) [reply .circleci/config.yml:19]: Good catch!

**Review by RichardDavies — APPROVED** [review]

**Review by joshuami — APPROVED** [review]

**Pull request merged**